src/chapters/html/structure.rst

```diff
@@ -0,0 +1,79 @@
+HTML Structure
+==============
+
+.. index:: ! head , ! title , ! html , ! doctype , ! body
+
+Programmers should follow certain rules about how to structure a HTML file.
+The rules about how to structure a HTML file and the tags used to lay out this structure are vital to the browser being able to render the page.
+
+Structure Rules
+---------------
+
+When it comes to laying out the overarching structure of a HTML file, a programmer should follow 5 rules:
+
+1. Every HTML file needs a ``DOCTYPE`` tag, specifying that the file is in fact, HTML. This is the only tag that does not require a closing tag.
```

Comment:
I don't like to be the "well actually" person, but I can't help myself.
- Should we say something about how DOCTYPE defines the version of HTML being used?
- With html5 <!DOCTYPE html> there are multiple elements that do not have closing tags `hr, img, input, link`